core/tests/test_utils.py

```diff
@@ -585,6 +585,20 @@ def logout(self):
         os.environ['USER_ID'] = ''
         os.environ['USER_IS_ADMIN'] = '0'
 
+    # pylint: disable=invalid-name
+    def assertStaticMethod(self, clazz, method_name):
```

Comment:
To me, it makes more sense to use the camelCase, because all the other assert methods use it. (`_assert_validation_error` is "private" just in this file so I can understand why it uses snake_case).

Reply:
Let's keep the camelCase for the reason that Vojta pointed.

Reply:
It would actually be better to avoid the pylint pragma, where possible. We already do this (search for self.assert_cannot_edit, self.assert_can_voiceover).

Could you please change this @vojtechjelinek? (And @kevinlee12, would changing this be OK? I think this can be taken as a general principle going forward, i.e. avoid adding pylint pragmas where possible.)